src/app/components/game-components/game-component/GameComponent.ts

```diff
@@ -219,6 +220,39 @@ export abstract class GameComponent<R extends SuperRules<M, S, C, L>,
         return this.getSVGTranslation(svgX, svgY);
     }
 
+    public getArrowTransform(boardWidth: number, boardHeight: number, orthogonal: Orthogonal): string {
+        // The triangle will be wrapped inside a square
```

Comment:
the triangle ? You mean "the arrow head"? ou "the triangle that forms the arrow head"